includes/class-wc-shipping-zones.php

```diff
@@ -55,6 +55,8 @@ public static function get_zone( $zone_id ) {
 	 * @return WC_Shipping_Zone|bool
 	 */
 	public static function get_zone_by( $by = 'zone_id', $id = 0 ) {
+                $zone_id = false ;
```

Comment:
Need fix the indentation here, we use spaces on WordPress and not spaces.
And need to remove that extra space before `;`.